http/oidc/src/main/java/org/wildfly/security/http/oidc/IDToken.java

```diff
@@ -184,15 +184,6 @@ public String getClaimsLocales() {
         return getClaimValueAsString(CLAIMS_LOCALES);
     }
 
-    /**
-     * Get the access token hash claim.
-     *
-     * @return the access token hash claim
-     */
-    public String getAccessTokenHash() {
```

Comment:
We don't need to remove this method. This method is different from the private `getAccessTokenHash` method in the `TokenValidator` file.